tests/aws/test_serverless.py

```diff
@@ -14,8 +15,11 @@ def get_base_dir() -> str:
     return os.path.join(os.path.dirname(__file__), "serverless")
 
 
+LOG = logging.getLogger(__name__)
+
+
 class TestServerless:
-    @pytest.fixture(scope="session")
+    @pytest.fixture(scope="class")
```

Comment:
nit: Why? There are no other tests in this file, and the fixture is only scoped to this class anyway.

Reply:
Because the fixture will not be cleaned up at the end of the class, even if it is only available (from a python standpoint) in there. A test run I had, did not perform the cleanup after the test class execution finished.

From the docs:
```
session: the fixture is destroyed at the end of the test session.
```